e2e/libs/keywords/workload_keywords.py

```diff
@@ -121,3 +124,21 @@ def wait_for_workload_claim_size_expanded(self, workload_name, claim_index=0):
         logging(f'Waiting for {workload_name} volume {volume_name} to expand to {expanded_size}')
         self.volume.wait_for_volume_expand_to_size(volume_name, expanded_size)
         self.volume.wait_for_volume_detached(volume_name)
+
+    def wait_for_workload_pods_kept_in_state_on_origin_node(self, workload_name, last_volume_node, expect_state, namespace="default"):
```

Comment:
Can we just wait and se if there is any workload pods stuck in the expected state, and return the pod information, then compare the pod node name with `last_volume_node`?

Like:
```
pod = wait_for_pod_kept_in_state(workload_name, expected_state)
Should Be Equal    pod.node    last_volume_node
or
Should Not Be Equal    pod.node    last_volume_node
```

Do you think it's feasible?

Reply:
Updated and test [report here](https://github.com/longhorn/longhorn-tests/files/15376596/report.zip)